app/assets/stylesheets/layouts/_application.scss

```diff
@@ -0,0 +1,4 @@
+.app-header {
```

Comment:
All layouts must be namespaced to allow other layouts to be used:

```
.layout-application {
  .app-header {

  }
}
```

That class must be added to the top most HTML element which is the HTML tag e.g., https://github.com/nimblehq/mia-web/blob/development/engines/mia_campaign/app/views/layouts/mia_campaign/application.html.slim#L2

Reply:
Thanks for the tip. Fixed in fa9c4364834e6c4e71fe3b17cf9b9a7f9574dd21